lib/cpu_gauge_field.cpp

```diff
@@ -65,8 +65,8 @@ namespace quda {
       }
 
       if (create == QUDA_NULL_FIELD_CREATE || create == QUDA_ZERO_FIELD_CREATE) {
-	gauge = (void **) safe_malloc(bytes);
-	if(create == QUDA_ZERO_FIELD_CREATE) memset(gauge, 0, bytes);
+	gauge = bytes ? (void **) safe_malloc(bytes) : nullptr;
+	if(create == QUDA_ZERO_FIELD_CREATE && bytes) memset(gauge, 0, bytes);
```

Comment:
Look like this PR needs a clang-format.

Reply:
Discussed offline, will be the last step before merging to make it easier to track commit histories.